src/docs/cookbook/forms/focus.md

```diff
@@ -9,22 +9,24 @@ next:
 ---
 
 When a text field is selected and accepting input, it is said to have "focus."
-Generally, users can focus text fields by tapping on them, and developers
-can focus text fields using the tools described in this recipe.
+Generally, users shift focus to a text field by tapping,
+and developers shift focus to a text field programmatically by
+using the tools described in this recipe.
 
 Managing focus is a fundamental tool for creating forms with an intuitive
-flow. For example, say we have a search screen with a text field. When
-the user navigates to the search screen, we can focus the search term text field.
+flow. For example, say you have a search screen with a text field.
+When the user navigates to the search screen,
+you can focus on the search term text field.
 This allows the user to start typing as soon as the screen
-is visible, without needing to manually tap on the text field!
+is visible, without needing to manually tap the text field.
 
-In this recipe, we'll learn how to focus a text field as soon as it's visible
-as well as how to focus a text field when a button is tapped.
+In this recipe, learn how to focus on a text field as soon as it's visible,
```

Comment:
"focus on" sounds wrong, throughout. "give the focus to " might be right.

https://webaim.org/techniques/keyboard/ uses "has keyboard focus", "focused element", and "receive keyboard focus", "lose focus", ... https://en.wikipedia.org/wiki/Focus_(computing) has "has the focus", "moving the focus", "getting the focus", "giving [an element]...the focus", "can receive focus", ... https://documentation.basis.com/BASISHelp/WebHelp/events/keyboard_focus_navigation_and_default_button.htm is the top results and might also be helpful.